packages/office-ui-fabric-react/src/components/Persona/Persona.tsx

```diff
@@ -122,7 +123,13 @@ export class Persona extends BaseComponent<IPersonaProps, {}> {
       <div className={ className }>
         { render
           ? render(this.props)
-          : <TooltipHost content={ text } overflowMode={ TooltipOverflowMode.Parent }>{ text }</TooltipHost>
+          : <TooltipHost
+            content={ text }
+            overflowMode={ TooltipOverflowMode.Parent }
+            directionalHint={ DirectionalHint.topLeftEdge }
```

Comment:
Does this work in RTL?

Reply:
In the ToolTip props documentation, it states that if directionalHintForRTL is not specified, a mirror of `directionalHint` will be used instead. I double checked the build, and it appears to be working as expected.
![image](https://user-images.githubusercontent.com/16619843/32506923-be94b49c-c39a-11e7-8db3-359b6dfdfa80.png)

Reply:
Cool, thanks!